### PR TITLE
ci: bump android apple dns match

### PR DESCRIPTION
**Pull request by jamilbk**
Bumps Android -> 1.3.4, Apple -> 1.3.5

### CODE REVIEW

**Comment by jamilbk** [comment]
Moving to draft because Apple is being slow to review the macOS app. Must have jinxed the Review Gods when making that video today.

Screenshot 2024-09-26 at 12 11 10 AM

File: website/src/components/Changelog/Apple.tsx [hunk 1]
```diff
@@ -11,13 +11,12 @@ export default function Apple() {
       title="macOS / iOS"
     >
       {/* When you cut a release, remove any solved issues from the "known issues" lists over in `client-apps`. This must not be done when the issue's PR merges. */}
-
-      <Unreleased>
```
Comment (thomaseizinger) [inline website/src/components/Changelog/Apple.tsx:15]: Missing new unreleased section.

**Review by thomaseizinger — APPROVED** [review]

File: website/src/components/Changelog/Apple.tsx [hunk 2]
```diff
@@ -11,13 +11,12 @@ export default function Apple() {
...
+      <Entry version="1.3.5" date={new Date("2024-09-26")}>
```
Comment (jamilbk) [inline website/src/components/Changelog/Apple.tsx:15]:
```suggestion
      <Unreleased></Unreleased>
      <Entry version="1.3.5" date={new Date("2024-09-26")}>
```

File: website/src/components/Changelog/Apple.tsx (hunk 1 above)
Comment (jamilbk) [reply website/src/components/Changelog/Apple.tsx:15]: Thanks, fixed

**Pull request merged**